include/openssl/rand.h

```diff
@@ -24,7 +24,8 @@ extern "C" {
 
 /* Random number generation. */
 
-/* RAND_bytes writes |len| bytes of random data to |buf| and returns one. */
+/* RAND_bytes writes |len| bytes of random data to |buf| and returns one on
```

Comment:
s/ and/. It/

The convention in the code is to have "It returns one..." as a separate sentence. Let's be consistent with that.